includes/class-wp-job-manager-post-types.php

```diff
@@ -584,8 +584,8 @@ public function job_feed() {
 			'ignore_sticky_posts' => 1,
 			'posts_per_page'      => $input_posts_per_page,
 			'paged'               => absint( get_query_var( 'paged', 1 ) ),
-			'tax_query'           => [],
-			'meta_query'          => [],
+			'tax_query'           => [], // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_tax_query -- Empty.
+			'meta_query'          => [], //// phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query -- Empty.
```

Comment:
I think we need just a single `//` right?

```suggestion
			'meta_query'          => [], // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_query -- Empty.
```